src/llmcompressor/observers/imatrix.py

```
@@ -20,13 +23,21 @@
-    Falls back to uniform MSE for global_scale search.
-    Extra observer_kwargs: maxshrink, patience, grid, norm, strict.
+    Falls back to uniform MSE when importance data is unavailable.
+
+    Importance is accumulated as raw ``_imatrix_sum`` / ``_imatrix_count``
+    and synced across DDP ranks via ``_sync_dict`` before observation.
     """
 
+    _act_sync_dict = {
+        "_imatrix_sum": dist.ReduceOp.SUM,
+        "_imatrix_count": dist.ReduceOp.SUM,
+    }
```

Ruff (0.15.11)

[warning] Mutable default value for class attribute

(RUF012)